tests/tck/features/match/Base.feature

```diff
@@ -15,6 +15,16 @@ Feature: Basic match
     Then the result should be, in any order, with relax comparison:
       | v            |
       | ("Yao Ming") |
+    When executing query:
+      """
+      MATCH (v:player) WHERE v.age < 0 RETURN v
```

Comment:
The filter conditions here will be replaced with `v.name is null` after [https://github.com/vesoft-inc/nebula-graph/pull/731](https://github.com/vesoft-inc/nebula-graph/pull/731) is merged.